### DISCUSSION TITLE
Issue with the nature renderer shader and Smoothness

### DISCUSSION THREAD

**Discussion by HellbourneAL** [visualdesigncafe/nature-renderer#16] [Help]
<img width="1004" alt="Screenshot 2024-02-19 083857" src="https://github.com/visualdesigncafe/nature-renderer/assets/160437334/2572c5da-e285-424a-a8d5-22594dcdc703">
<img width="1003" alt="Screenshot 2024-02-19 083919" src="https://github.com/visualdesigncafe/nature-renderer/assets/160437334/8725909e-ae84-47f0-8544-eb4c89153c7e">
So when the shader patches, it gives the material a glossy look, when reverting the non-patched original shader of the object its back to normal, issue is the new shader that is patched by nature renderer seems to ignore all metallic/smoothness settings, they seem to have no effect on the material, thus the metarial remains reflective/shiny.

Is there a way to fix this for the patched materials/shaders?

**Comment by maxplooi** [comment]
Thank you for reporting this, it is a very strange issue since Nature Renderer does not modify any of the shader's features, it just adds a few additional lines of code for Nature Renderer. If you have experience with editing shaders then could you try to manually patch the shaders by following the [guide](https://visualdesigncafe.com/nature-renderer/docs/2022/custom-shaders)? If you have no experience with this, then could you upload the shader here or via the [business inquiry form](https://support.visualdesigncafe.com/hc/en-us/requests/new)? Then I'll take a look at it for you.
